static/js/public/fullRecord.js

```diff
@@ -115,20 +112,24 @@ define('fullRecord', ['module', 'jquery', 'JP2Viewer', 'StructureView', 'dataTab
 				return img
 				}, targets: 0
 			},
-			{ render: function (data, type, row) { return '<a href="/record/' + row.id + '" + aria-label="View ' + row.title +'">' +row.title + '</a>'; }, targets: 1 },
+			{ render: function (data, type, row) { return '<a href="/record/' + row.id + '" aria-label="View ' + row.title +'">' +row.title + '</a>'; }, targets: 1 },
 			{ render: function (data, type, row) { return getFileType(row); }, targets: 2 },
 			{ render: function (data, type, row) { return getOriginalFileValue(row.datastream, 'file_size');  }, targets: 3 },
 			{ render: function (data, type, row) { return '<a href="/record/' + row.id + '" aria-label="View ' + row.title +'">' +
 					'<i class="fa fa-search-plus is-icon"' + ' title="View"></a>'; },
 				targets: 4
 			},
-			{ render: function (data, type, row) { return '<a href="/indexablecontent/' + row.id + '?dl=true" aria-label="Download ' + row.title +'">' +
-					'<i class="fa fa-download is-icon" title="Download"></a>'; },
+			{ render: function (data, type, row) {
+					if (row.permissions.indexOf('viewOriginal') === -1) {
+						return '<i class="fa fa-download is-icon no-download" title="Download Unavailable">';
```

Comment:
I know the browser corrects this, but shouldn't this have a closing `</i>`? Looks like on the font-awesome site has it that way, and looks similar to the error listed here https://html.spec.whatwg.org/#syntax-errors labeled as "Errors that result in disproportionately poor performance". Looks like this is happening in a few other places in this file.